choco/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,26 @@
+$ErrorActionPreference = 'Stop';
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$fileLocation = Join-Path $toolsDir 'testkube_1.3.25_Windows_i386.msi'
+
+$packageArgs = @{
+  packageName   = $env:ChocolateyPackageName
+  unzipLocation = $toolsDir
+  fileType      = 'MSI'
+  file         = $fileLocation
+
+  softwareName  = 'testkube'
+
+  checksum      = '28EA0DE9642290B7D70E0610D3507D3A4145A6413E585DE8C7EBC4C1018B9B34'
+  checksumType  = 'sha256'
+
+  silentArgs    = "/qn /norestart /l*v `"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+  validExitCodes= @(0, 3010, 1641)
+}
+
+Install-ChocolateyInstallPackage @packageArgs
+
```

Comment:
Are these inteneded empty lines?

Reply:
no :) thanks for noticing, removed them.